src/app/home/superset/superset.service.ts

```diff
@@ -0,0 +1,119 @@
+/** Angular Imports */
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+/** rxjs Imports */
+import { Observable, throwError } from 'rxjs';
+import { catchError, switchMap } from 'rxjs/operators';
+
+/** EmbedDashboard SDK import */
+import { embedDashboard } from '@superset-ui/embedded-sdk';
+
+/** 
+ * Superset Service
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class SupersetService {
+  
+  /**
+   * API URL of Superset to send request
+   */
+  private apiUrl = 'http://localhost:8088/api/v1/security';
+
+  /**
+   * @param {HttpClient} http Http Client to send requests.
+   */
+  constructor(private http: HttpClient) { }
+
+  /**
+   * 
+   * @returns { access token }
+   */
+  private fetchAccessToken(): Observable<any> {
+    const body = {
+      "username": "admin",
```

Comment:
I am not sure it is a good idea to store credential information at UI side... anybody can read them and abuse...